docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Is this to support Windows?

Reply:
I don't know, it's autogenerated by sphinx